### PR TITLE
Core: Player name property on world class

### PR DESCRIPTION
**Pull request by NewSoupVi**
`self.multiworld.get_player_name(self.player)` is so stupid lol

Edit: Overcooked was already doing this itself so I needed to make it not do that. @toasterparty

### CODE REVIEW

**Comment by NewSoupVi** [comment]
Oh god damnit

File: worlds/AutoWorld.py [hunk 1]
```diff
@@ -468,6 +468,10 @@ def get_entrance(self, entrance_name: str) -> "Entrance":
     def get_region(self, region_name: str) -> "Region":
         return self.multiworld.get_region(region_name, self.player)
 
+    @property
+    def player_name(self):
```
Comment (beauxq) [inline worlds/AutoWorld.py:472]:
```suggestion
    def player_name(self) -> str:
```

Comment (NewSoupVi) [reply worlds/AutoWorld.py:472]: o o p s

**Review by toasterparty — APPROVED** [review]

**Review by hatkirby — APPROVED** [review]
Simple and self explanatory. Lgtm.

**Review by ScipioWright — APPROVED** [review]
LGTM, very simple

**Review by Exempt-Medic — APPROVED** [review]
LGTM and it works

**Review by black-sliver — APPROVED** [review]
Approved by world devs and I see no reason against adding it.

**Pull request merged**